docs/configuration.json

```diff
@@ -55,57 +55,6 @@
       }
     ]
   },
-  {
```

Comment:
Time to get rid of test (and outdated) networks from the main configuration file.